mono/eglib/eglib-remap.h

```diff
@@ -94,7 +94,6 @@
 #define g_list_length monoeg_g_list_length
 #define g_list_nth monoeg_g_list_nth
 #define g_list_nth_data monoeg_g_list_nth_data
-#define g_list_prepend monoeg_g_list_prepend
```

Comment:
I think you meant to get rid of `g_string_prepend`?

Reply:
Right oops thank you.